src/components/fastfood/fastfood.html

```diff
@@ -0,0 +1,8 @@
+<div class="fastfood">
+  <article class="fastfood__readnow">
+    <div class="fastfood__article">ARTICLE</div>
+    <div>The pros and cons of fast food.</div>
+    <a href="#"><img src="../../assets/readnow.png" alt="read now button" /></a>
+  </article>
+  <img src="../../assets/burger.png" alt="burger" />
+</div>
```

Comment:
I would change `div` and `article`. The `img` is part of the `article`.
```suggestion
<article class="fastfood">
  <div class="fastfood__readnow">
    <div class="fastfood__article">ARTICLE</div>
    <div>The pros and cons of fast food.</div>
    <a href="#"><img src="../../assets/readnow.png" alt="read now button" /></a>
  </div>
  <img src="../../assets/burger.png" alt="burger" />
</article>
```